pom.xml

```diff
@@ -146,6 +152,12 @@
             <version>${jstl.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
Hi this dependency has already been added to pom XML, please remove this dependency